src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp

```diff
@@ -685,12 +685,14 @@ class boss_stormcaller_brundir : public CreatureScript
 
             if (IsEncounterComplete(pInstance, me))
             {
+                me->NearTeleportTo(me->GetPositionX(), me->GetPositionY(), 427.5, me->GetOrientation());
                 pInstance->SetData(TYPE_ASSEMBLY, DONE);
                 me->CastSpell(me, 65195, true); // credit
                 Talk(SAY_BRUNDIR_ENCOUNTER_DEFEATED);
             }
             else
             {
+                me->NearTeleportTo(me->GetPositionX(), me->GetPositionY(), 427.5, me->GetOrientation());
```

Comment:
why placing the same line twice, in the `if` and `else` block? we can just put it once, before the `if` block

Reply:
edited.